layouts/partials/plugin/analytics.html

```diff
@@ -48,4 +48,38 @@
         <script defer src='https://static.cloudflareinsights.com/beacon.min.js' data-cf-beacon='{"token": "{{ $analytics.cloudflare.token }}"}'></script>
     {{- end -}}
 
+    {{- /* Splitbee Analytics */ -}}
+    {{- if $analytics.splitbee.enable -}}
+        {{- if $analytics.splitbee.data_token -}}
+            {{- if $analytics.splitbee.No_cookie -}}
+                {{- if $analytics.splitbee.Do_not_track -}}
+                    <script data-token="{{ $analytics.splitbee.data_token }}" data-respect-dnt data-no-cookie defer src="https://cdn.splitbee.io/sb.js"></script>
+                {{- else -}}
+                    <script data-token="{{ $analytics.splitbee.data_token }}" data-no-cookie defer src="https://cdn.splitbee.io/sb.js"></script>
+                {{- end -}}
+            {{- else -}}
+                {{- if $analytics.splitbee.Do_not_track -}}
+                    <script data-token="{{ $analytics.splitbee.data_token }}" data-respect-dnt defer src="https://cdn.splitbee.io/sb.js"></script>
+                {{- else -}}
+                    <script data-token="{{ $analytics.splitbee.data_token }}" defer src="https://cdn.splitbee.io/sb.js"></script>
+                {{- end -}}
+            {{- end -}}
+        {{- else -}}
+            {{- if $analytics.splitbee.No_cookie -}}
+                {{- if $analytics.splitbee.Do_not_track -}}
+                    <script data-respect-dnt data-no-cookie defer src="https://cdn.splitbee.io/sb.js"></script>
+                {{- else -}}
+                    <script data-no-cookie defer src="https://cdn.splitbee.io/sb.js"></script>
+                {{- end -}}
+            {{- else -}}
+                {{- if $analytics.splitbee.Do_not_track -}}
+                    <script data-respect-dnt defer src="https://cdn.splitbee.io/sb.js"></script>
+                {{- else -}}
+                    <script defer src="https://cdn.splitbee.io/sb.js"></script>
+                {{- end -}}
+            {{- end -}}
+        {{- end -}}
+
+    {{- end -}}
```

Comment:
I am not particular satisfying with how you add configs with nested `if`s. 

Maybe you can do something like this

https://github.com/HEIGE-PCloud/DoIt/blob/98107d20dd745635a486fbb906a09e826f19926e/layouts/partials/comment.html#L150-L171

Reply:
I have changed the style to a [more elegant one](https://github.com/HEIGE-PCloud/DoIt/pull/793/commits/61fc184e93dd7c58e97ef49e96da7926f03c6f4c).

btw learn a lot as a Hugo newbie, thx.